models/exprensive.model.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+
+const ExpensiveSchema = new Schema({
+
+    ExpensiveID:Number,
+    ExpensiveType:String,
+    Amount:Number,
+    Date:Date,
+    UserID:Number
```

Comment:
UserID eke type eka wenas wenna ona bn eka refference ekak enna ona user model eken
Like this,
UserID :{
        type: mongoose.Schema.Types.ObjectId,
        ref: 'user'
    },